src/loaders/FileLoader.js

```diff
@@ -173,7 +179,29 @@ class FileLoader extends Loader {
 
 					default:
 
-						return response.text();
+						return response.arrayBuffer().then( ab => {
+
+							let label = 'utf-8';
+							if ( this.mimeType !== undefined ) {
+
+								const i = this.mimeType.indexOf( '=' );
+								if ( ~ i ) {
```

Comment:
You can please rewrite this line so `~` isn't used. This will make it easier to read.

Reply:
no problem